cli_tools_cloudbuild.yaml

```diff
@@ -89,6 +89,10 @@ steps:
   args: 
   - --destination=gcr.io/$PROJECT_ID/gce_vm_image_import:$_RELEASE
   - --destination=gcr.io/$PROJECT_ID/gce_vm_image_import:$COMMIT_SHA
+  - --destination=us-central1-docker.pkg.dev/$PROJECT_ID/gce_vm_image_import:$_RELEASE
```

Comment:
Looking forward, how will this look on the consumer side? For example, in [1], the step uses `gcr.io/compute-image-tools/gce_vm_image_import:release`. Will the docs get updated to show each regional option? I'm curious about gcloud as well -- would gcloud pick the regional artifact based on the specified region/zone?

1. https://cloud.google.com/compute/docs/import/importing-virtual-disks#bootable